.travis.yml

```diff
@@ -32,13 +32,11 @@ matrix:
     - os: osx
       osx_image: xcode9.1
 
-cache:
-  directories:
-    - ui/node_modules
-
```

Comment:
I'll remove this before merging. It's only here to make sure other branches don't try using newer node modules.